src/DetailsView/components/load-assessment-helper.ts

```diff
@@ -9,18 +10,31 @@ export class LoadAssessmentHelper {
         private readonly detailsViewActionMessageCreator: DetailsViewActionMessageCreator,
         private readonly fileReader: FileReader,
         private readonly document: Document,
-        private readonly tabId: number,
     ) {}
 
-    public getAssessmentForLoad() {
+    public getAssessmentForLoad(
+        setAssessmentState: Function,
```

Comment:
We should use appropriate types for these functions instead of the `Function` type.

Reply:
addressed this.